packages/design-system/src/components/Modal/Modal.scss

```
@@ -0,0 +1,99 @@
+@use '~@talend/design-tokens/lib/tokens';
+
+$modal-backdrop-z-index: 100;
+$modal-z-index: $modal-backdrop-z-index + 1;
+
+.modal-backdrop {
+	background-color: tokens.$coral-color-assistive-background;
+	opacity: tokens.$coral-opacity-l;
+	position: absolute;
+	top: 0;
+	bottom: 0;
+	left: 0;
+	right: 0;
+	z-index: $modal-backdrop-z-index;
+}
+
+.modal {
+	position: absolute;
```

```suggestion
	position: fixed;
```
With this you can use percentages instead of "vw" and "vh". "vh" specifically is a bitch.